apps/pigment-css-next-app/next.config.js

```diff
@@ -118,13 +120,7 @@ const pigmentOptions = {
       ...context,
       require: (id) => {
         if (id === '@mui/styled-engine' || id === '@mui/styled-engine-sc') {
-          return {
```

Comment:
This is not needed here. I'll clean this up separately since it's not a package.